packages/backend/src/services/note/reaction/create.ts

```diff
@@ -124,7 +124,16 @@ export default async (user: { id: User['id']; host: User['host']; }, note: Note,
 			const reactee = await Users.findOne(note.userId);
 			dm.addDirectRecipe(reactee as IRemoteUser);
 		}
-		dm.addFollowersRecipe();
+
+		if (['public', 'home', 'followers'].includes(note.visibility)) {
+			dm.addFollowersRecipe();
+		} else if (note.visibility === 'specified') {
+			const visibleUsers = await Promise.all(note.visibleUserIds.map(id => Users.findOne(id)));
```

Comment:
Misskey has been migrated to TypeORM 3.0. In 3.0, you need to write as follows: `Users.findOneBy({ id })`

Reply:
Oh yea, forgot about that, still haven't pulled that to my instance yet

Reply:
I think I forgot to pull the up to date version so I got confused by code still using the old way. Has been fixed now.